quickstart/README.md

```diff
@@ -0,0 +1,54 @@
+# Quarkus demo: Cassandra Client
+
+This example showcases how to use the Cassandra client with Quarkus. 
+
+# Cassandra instance - Running with Docker
+
+Just run it as follows:
+`docker run \
+   --name local-cassandra-instance \
+   -p 7000:7000 \
+   -p 7001:7001 \
+   -p 7199:7199 \
+   -p 9042:9042 \
+   -p 9160:9160 \
+   -p 9404:9404 \
+   -d \
+   launcher.gcr.io/google/cassandra3`
```

Comment:
We don't _need_ to expose all those ports to vet out the quickstart, right?  If we get 9042 we should be okay.

Maybe just add a comment here saying only 9042 is required and the others are optional?

Reply:
I wanted to give the users full exposed Cassandra so I can connect using CQLSH, JMX, driver and so one.